packages/cli/src/metrics/prometheus-metrics.service.ts

```diff
@@ -24,10 +23,10 @@ export class PrometheusMetricsService {
 
 	private readonly prefix = config.getEnv('endpoints.metrics.prefix');
 
-	private readonly includes = {
+	private readonly includes: Includes = {
 		metrics: {
 			default: config.getEnv('endpoints.metrics.includeDefaultMetrics'),
-			api: config.getEnv('endpoints.metrics.includeApiEndpoints'),
```

Comment:
is this a breaking change 🤔

Reply:
How do you mean? This is an internal implementation detail, the schema path and the env remain the same.

Reply:
does this not change the metric name that gets scraped by prometheus?

Reply:
Oh that's what you mean. Since we advertise this as [experimental](https://docs.n8n.io/hosting/configuration/configuration-examples/prometheus/) I take it this is okay to change, but I can add a mention just in case.